### PR TITLE
Added radio buttons to select isSudo true/false for script task.

### PR DESCRIPTION
**Pull request by divakarkonakalla**
@udeshrl , Please review the changes and merge.

### CODE REVIEW

File: client/cat3/src/partials/sections/dashboard/workzone/orchestration/popups/newTaskCtrl.js [hunk 1]
```diff
@@ -42,6 +42,13 @@
 			$scope.optionScriptToggled = function() {
 				$scope.isAllScriptSelected = $scope.scriptTaskList.every(function(itm){ return  itm._isScriptSelected; })
 			}
+			$scope.isSudoSelected = function(val) {
+				if(val == "true") {
```
Comment (udeshrl) [inline client/cat3/src/partials/sections/dashboard/workzone/orchestration/popups/newTaskCtrl.js]: I don't think you need this function anymore. Please try to remove this and see if it still works fine

Comment (divakarkonakalla) [inline client/cat3/src/partials/sections/dashboard/workzone/orchestration/popups/newTaskCtrl.js]: @hrushikesh07 , Please review the changes and merge.

File: client/cat3/src/partials/sections/dashboard/workzone/orchestration/popups/newTask.html [hunk 2]
```diff
@@ -80,18 +80,30 @@ <h4 class="modal-title" ng-if="isEditMode">
...
-										<option value="">Select</option>
-										<option value="Bash">Bash</option>
-									</select>
-								</span>
+								</div>
+								<div class="col-md-6 col-sm-6 col-xs-6 margintop15">
+									<label class="control-label">Run as Sudo:</label>
+									<br />
+									<div class="paddingtop8">
+										<input type="radio" ng-value="true" ng-model="isSudo" ng-click="isSudoSelected('true');" />
```
Comment (hrushikesh07) [inline client/cat3/src/partials/sections/dashboard/workzone/orchestration/popups/newTask.html]: Pass the boolean value

Comment (hrushikesh07) [inline client/cat3/src/partials/sections/dashboard/workzone/orchestration/popups/newTask.html]: remove the ng-click

File: client/cat3/src/partials/sections/dashboard/workzone/orchestration/popups/newTaskCtrl.js (hunk 1 above)
Comment (hrushikesh07) [inline client/cat3/src/partials/sections/dashboard/workzone/orchestration/popups/newTaskCtrl.js]: Remove the "isSudoSelected" function .

**Pull request merged**